<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="CESIT-LAP-0420" failures="2" tests="7" name="MyTest" time="15.179" errors="0" timestamp="2023-10-30T18:42:38 IST">
  <testcase classname="testCases.sample" name="sample1" time="1.38">
    <skipped/>
  </testcase> <!-- sample1 -->
  <testcase classname="testCases.sample" name="sample2" time="1.379">
    <skipped/>
  </testcase> <!-- sample2 -->
  <testcase classname="testCases.sample" name="sample2" time="5.223">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.sample.sample2(sample.java:25)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- sample2 -->
  <testcase classname="testCases.sample" name="sample2" time="8.15">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.sample.sample2(sample.java:25)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- sample2 -->
  <testcase classname="testCases.sample" name="sample1" time="11.843"/>
  <testcase classname="testCases.sample" name="sample3" time="2.816"/>
  <testcase classname="testCases.sample" name="sample1" time="12.813"/>
</testsuite> <!-- MyTest -->
